go/vt/vtgate/planbuilder/jointree.go

```diff
@@ -77,10 +77,15 @@ type (
 		leftJoins []*outerTable
 
 		// these fields are set if a vindex will be used for this route
-		currentCost      cost // currentCost tracks the cost of the chosen access method
-		vindex           vindexes.Vindex
-		vindexValues     []sqltypes.PlanValue
+		currentCost cost // currentCost tracks the cost of the chosen access method
```

Comment:
we could collect all the vindex selections into a common struct, it is clear that they belong together, and to make it easy to clear out the selected opcode and vindex in `reset()`

Reply:
in the next PR